src/actions/users-me.js

```diff
@@ -6,11 +6,14 @@ import * as backend from "../services/backend";
 
 export const retryIfUnauthorized = (func, dispatch) => {
   return (...args) => {
-    return func(...args).catch((err) => {
-      if (dispatch && err instanceof HttpError && err.response.status === 401) {
+      return func(...args).catch((err) => {
+      if (dispatch && err instanceof HttpError && (err.response.status === 401 || err.response.status === 403)) {
```

Comment:
We should IMO fix backend to return `401` instead of patching FE. What scenario you are trying to fix here?

Could you fix it on backend or add bug to project backlog, please?